src/components/sandboxLink.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import usePlatform from "./hooks/usePlatform";
+
+const scenarios = [
+    'performance',
+    'releases',
+    'alerts',
+    'discover',
+    'dashboards',
+    'projects',
+    'oneDiscoverQuery',
+    'oneIssue',
+    'oneBreadcrumb',
+    'oneStackTrace',
+    'oneTransaction',
+    'oneWebVitals',
+    'oneTransactionSummary',
+    'oneRelease',
+  ] as const;
+
+type Props = {
+    children: React.ReactNode;
+    scenario?: typeof scenarios[number];
+    projectSlug?: string;
```

Comment:
@Neo-Zhixing wondering if we should be more explicit with our types here too to prevent someone from using an invalid project

Reply:
In the end it doesn't matter, I don't think you have any type checking when using the component from a `mdx` file